Sources/FontRenderer+singleLineSize.swift

```diff
@@ -116,15 +116,15 @@ extension FontRenderer {
             }
 
             guard Find_Glyph(self.rawPointer, characterCode, CACHED_METRICS) == 0 else {
-                assertionFailure("Couldn't find glyph")
+                print("Glyph \(characterCode) ('\(Character(UnicodeScalar(characterCode)!))') could not be found")
```

Comment:
are you sure we don't want to crash in devel when this happens? It may get lost

Reply:
Good point. I'm fine to make these assert again now that we've fixed this issue. Nicely spotted, thanks @michaelknoch